presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergUtil.java

```diff
@@ -121,4 +143,36 @@ private static boolean isSnapshot(Table icebergTable, Long id)
         return stream(icebergTable.snapshots())
                 .anyMatch(snapshot -> snapshot.snapshotId() == id);
     }
+
+    private static Table loadMetadataTable(MetadataTableType type, SchemaTableName table, HiveMetastore metastore, HdfsEnvironment hdfsEnvironment, HdfsContext hdfsContext, HiveIdentity identity)
+    {
+        if (type != null) {
```

Comment:
We don't need to check since this method isn't called with null. It's the caller's responsibility to do the check.